File: Blackjack21-JAVA/Blackjack21Library/src/Blackjack21/Game/Exceptions/DeckEmptyException.java
package Blackjack21.Game.Exceptions;

/**
 * Throws a deck not found exception
 */
public class DeckEmptyException extends Exception
{

    public DeckEmptyException()
    {
        super("There are no cards in the deck");
    }

    public DeckEmptyException(String message)
    {
        super(message);
    }

}

File: BlackJack-CSHARP/Blackjack21.Game/Game/Exceptions/DeckEmptyException.cs
using System;

namespace Blackjack21.Game.Exceptions
{
    /// <summary>
    /// Throws a deck not found exception
    /// </summary>
    public class DeckEmptyException : Exception
    {

        public DeckEmptyException(string message = "There are no cards in the deck") : base(message)
        {

        }
    }
}
